trac-env/htdocs/css/_noshadows.scss

```diff
@@ -0,0 +1,114 @@
+// Friends don't let friends use box/text-shadow...
```

Comment:
I'd like if you included a better explanation, lest we imply the developers of Trac can't be our friends. :jack_o_lantern: